src/owncloud/apache_config

```diff
@@ -14,3 +15,14 @@ Alias "/apps" "${SNAP_DATA}/owncloud/apps"
 <IfModule dir_module>
     DirectoryIndex index.html index.php
 </IfModule>
+
+# MPM Event
+<IfModule event.c>
+	StartServers			2
+	MinSpareThreads			4
+	MaxSpareThreads			32
+	ThreadsPerChild			16
+	ServerLimit				4
+	MaxRequestWorkers		64
+	MaxConnectionsPerChild	0
+</IfModule>
```

Comment:
Okay. Just try to keep configs pretty general in this repo-- we can really tune it for the rpi in the fork. I'll trust you here, since I know nothing about these values :wink: .